apps/client/src/app/modules/list/list-details-panel/list-details-panel.component.ts

```diff
@@ -455,6 +488,30 @@ export class ListDetailsPanelComponent implements OnChanges, OnInit {
     this.message.success(this.translate.instant('LIST.Copied_as_text'));
   }
 
+  public copyJSONExport() {
+    let rows: ListRow[];
+    if (this.tiers) {
+      rows = this.tiers.reduce((res, tier) => {
+        return [...res, ...tier];
+      }, []);
+    } else {
+      rows = this.displayRow.rows;
+    };
+
+    if (this._clipboardService.copyFromContent(
+      JSON.stringify(this.getSerializedRowData(rows))//,null,2)
+    ))
+      this.jsonCopied();
+  }
```

Comment:
Using the [cdk directive](https://material.angular.io/cdk/clipboard/overview) here would be better, as it's doing exactly this but it's maintained without us having to do anything.